### DISCUSSION TITLE
What goes in the "ChannelDefinition.QuantityName" field?

### DISCUSSION THREAD

**Discussion by SRM256** [GridProtectionAlliance/gsf#86] [Q&A] [answered]
The ChannelDefinition has a Name field and a QuantityMeasured field. I get what those are for. I'm not sure how QuantityName fits into the picture. Do you have an example where the name of the quantity doesn't just repeat those other two fields?

**Comment by StephenCWills** [comment, marked as answer]
I don't ever recall seeing a PQDIF file that uses that tag so I'm not aware of any industry standard that could be used to determine what purpose it serves. I agree that it seems a bit redundant next to the channel name.

This is pure speculation, but perhaps the idea is to have the quantity name disambiguate similar quantities that can't easily be differentiated otherwise whereas the channel name would allow you to disambiguate channels even if they are measuring the same quantities. For example, if two quantities differed in how they were sampled for aggregation, that information could be included in the quantity name. Then, if you are also taking the same type of measurement on both the supply side and the load side of a breaker, that information could be included in the channel name.

Even so, it's generally not a good idea to rely on free-form fields in automated analysis, and any human would be able to figure it out from an appropriately detailed channel name so I would say the quantity name field doesn't really provide much value in either case. 🤷
